lib/index.js

```diff
@@ -227,7 +227,8 @@ analyzer.prototype = {
 						parsedSelector = slickParse(selector)[0];
 
 						if (typeof parsedSelector === 'undefined') {
-							throw 'Unable to parse "' + selector + '" selector';
+							var positionDump = "Rule position start: " + rule.position.start.line + ',' + rule.position.start.column + "; end: " + rule.position.end.line + ',' + rule.position.end.column;
```

Comment:
Let's be consistent and use the following convention when printing line numbers and columns:

```js
'start @ ' + position.line + ':' + position.column
```